website/content/docs/api-gateway.mdx

```diff
@@ -0,0 +1,171 @@
+---
+layout: docs
+page_title: API Gateway
+description: Using Consul API gateway functionality
+---
+
+# Consul API Gateway
+
+This topic describes how to use the Consul API Gateway module, which enables applications and services running in a datacenter to be accessed by network clients outside the datacenter. This type of network traffic is commonly referred to as "north-south" network traffic.
+
+## Introduction
+
+Consul API Gateway is an implementation of the Kubernetes Gateway [API Specification](https://gateway-api.sigs.k8s.io/). The specification defines a set of custom resource definitions (CRD) that can create logical gateways and routes.
+
+## Requirements
+
+Your datacenter must meet the following requirements prior to configuring the Consul API Gateway:
+
+- A Kubernetes cluster must be running
+- Consul 1.11.1+
+
+## Installation
+
+1. Create a values file for your Consul server agents that contains the following parameters:
+
+   ```yaml
+   global:
+     name: consul
+     image: 'hashicorp/consul:1.11.1'
+     tls:
+       enabled: true
+   connectInject:
+     enabled: true
+   controller:
+     enabled: true
+   ```
+
+1. Install Consul API Gateway using the standard Consul Helm chart and specify the custom values file.
+
+```shell-session
+$ helm install consul hashicorp/consul --version 0.39.0 -f values.yaml
+```
+
+The following components will be installed: - Gateway controller - CRDs required by the Kubernetes Gateway API specification - `kustomize` manifests for completing the installation
+
+1. After `helm` installs Consul API Gateway packages, issue the following commands to apply the API gateway to your Kubernetes cluster.
+
+```shell-session
+$ kubectl apply -k "github.com/hashicorp/consul-api-gateway/config/crd?ref=v0.1.0-techpreview"
+$ kubectl apply -k "github.com/hashicorp/consul-api-gateway/config?ref=v0.1.0-techpreview"
+```
+
+## Usage
+
+1. Verify that the [requirements](#requirements) have been met.
+1. Verify that the Consul API Gateway software has been installed and applied (see [Installation](#installation)).
+1. Configure the gateway, listener(s), and route(s) as described in [Configuration](#configuration)
+1. Issue the `kubectl apply` command to implement the configurations, e.g.:
+
+```shell-session
+$ kubectl apply -f gateway-configuration.yaml
+```
+
+### Using the Consul API Gateway Binary
+
+You can manually start the Consul API Gateway control plane server using the binary located in the home directory to issue the `server` command:
```

Comment:
Your other steps do a great job of walking through the creation then usage of YAML files, but this one has been confused cuz I've never heard of the Consul API Gateway Binary up until this point. It looks like we're running some kind of script from a home directory, but I don't have the script or understand where or why I'm issuing this command.

Reply:
Thanks, I'll try to make this clearer up front